src/test/java/apimatic/core/utilities/CoreHelperTest.java

```diff
@@ -137,6 +137,24 @@ public void testBase64Encoding() {
         assertEquals(actualEncodedString, expectedEncodedString);
     }
 
+    @Test
+    public void testBase64EncodingWithNullValue() {
+        String username = null;
+        String password = "password";
+
+        String actualEncodedString = CoreHelper.getBase64EncodedCredentials(username, password);
+        assertEquals(null, actualEncodedString);
+    }
+
+    @Test
+    public void testBase64EncodingWithEmptyValue() {
```

Comment:
Blocker: Add assertions when the password is empty and when both credentials are empty.

Reply:
done